target/linux/sunxi/patches-5.4/302-sunxi-h5-add-support-for-nanopi-r1s.patch

```diff
@@ -0,0 +1,296 @@
+--- a/Documentation/devicetree/bindings/arm/sunxi.yaml
++++ b/Documentation/devicetree/bindings/arm/sunxi.yaml
```

Comment:
Please export  both patches from upstream Linux and add it with the commit header from here:
https://github.com/torvalds/linux/commit/9962cb9be2db877c232aaf00db40125c0d7bf4bc
https://github.com/torvalds/linux/commit/92ed3675574723a963152abbbe527b47f659340f
`git format-patch 9962cb9be2db877c232aaf00db40125c0d7bf4bc -1 `
`git format-patch 92ed3675574723a963152abbbe527b47f659340f -1 `

Reply:
better now ?

Reply:
this looks good

Reply:
please note that https://github.com/torvalds/linux/commit/9962cb9be2db877c232aaf00db40125c0d7bf4bc and https://github.com/torvalds/linux/commit/92ed3675574723a963152abbbe527b47f659340f doesn;t apply cleanly on 5.4 or .510 so I had to do some changes. I've kept the commit header as requested though